spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -393,10 +403,13 @@ def main(argv: Sequence[str]):
     fname_pmj = arguments.pmj
     distance_pmj = arguments.pmj_distance
     extent_pmj = arguments.pmj_extent
+    normalize_pam50 = arguments.normalize_PAM50
     path_qc = arguments.qc
     qc_dataset = arguments.qc_dataset
     qc_subject = arguments.qc_subject
 
+    if normalize_pam50 and (fname_vert_level is None or not perslice):
+        parser.error("Option '-normalize PAM50' requires options '-vertfile' and '-perslice 1'.")
```

Comment:
It might be good to split this into two different errors?

- for `fname_vert_level`, we require `-vertfile` to _exist_, so it might be better to move this part of the error into the section where we check for file existence, then word it similar to the `-vert` warning:

  https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/cb7cd19eb9478a0a7301374e997f8c22768bc2e3/spinalcordtoolbox/scripts/sct_process_segmentation.py#L393-L394

  "The vertebral level file must exist to use `-normalize-PAM50`."

- For `perslice`, we can keep the existing check.

Reply:
Also, small nit, but this still says `-normalize PAM50`.

Reply:
don in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/d7c5cf5f83151b647a060e3801296f1b33a56ea2